dndtroops-persistence/src/main/java/cz/muni/fi/pa165/dndtroops/entities/Troop.java

```diff
@@ -18,12 +19,17 @@
     @GeneratedValue (strategy = GenerationType.IDENTITY)
     private Long id;
 
-    @Column(unique = true)
+    @NotNull
+    @Column(nullable = false, unique = true)
     private String name;
 
+    @NotNull
+    @Column (nullable = false)
     private String mission;
```

Comment:
Can in D&D world exists a troop with no mission? I don't know, but I would say that Yes, it can. Just think about it and then write me a reply with your final decision.